services/ingestion/poller/YrPoller/src/YrPoller/yr_poller.py

```diff
@@ -51,11 +50,10 @@ def timestring_to_posix(time):
         new_time = datetime.strptime(time, "%Y-%m-%dT%H:%M:%S")  # + utx_delta
         return int(new_time.timestamp())
 
-    location_name = data.get("weatherdata", {}).get("location", {}).get(
-        "name", {})  # Will return {} if non-existing
-    forecasts = data.get("weatherdata", {}).get("forecast", {}).get(
-        "tabular", {}).get("time", {})  # {} if non-existing
-    ret = []
+    location_name = data.get("weatherdata", {}).get("location", {}).get("name", {})  # Will return {} if non-existing
+    forecasts = data.get("weatherdata", {}).get("forecast", {}).get("tabular", {}).get("time", {})  # {} if non-existing
+    ret = {"metadata": {"timestamp": datetime.now().timestamp()}, "data": {}}
```

Comment:
Mulig å caste denne til en Int?
```suggestion
    ret = {"metadata": {"timestamp": int(datetime.now().timestamp())}, "data": {}}
```